junit-jupiter-api/src/main/java/org/junit/jupiter/api/timezone/DefaultTimeZone.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2015-2025 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api.timezone;
```

Comment:
We discussed the package names in the last team call and would like to stick with a single `org.junit.jupiter.api.util` package for both extensions.

Reply:
Moved